src/create_helpers.ts

```diff
@@ -184,76 +207,79 @@ export function createClassMethod(doclet: IFunctionDoclet): ts.MethodDeclaration
     if (doclet.name.startsWith('exports.'))
         doclet.name = doclet.name.replace('exports.', '');
 
-    return handleComment(doclet, ts.createMethod(
-        undefined,      // decorators
-        mods,           // modifiers
-        undefined,      // asteriskToken
-        doclet.name,    // name
-        undefined,      // questionToken
-        typeParams,     // typeParameters
-        params,         // parameters
-        type,           // type
-        undefined       // body
-    ));
+    return handleComment(
+        doclet,
+        ts.createMethod(
+            undefined, // decorators
+            mods, // modifiers
+            undefined, // asteriskToken
+            doclet.name, // name
+            undefined, // questionToken
+            typeParams, // typeParameters
+            params, // parameters
+            type, // type
+            undefined, // body
+        ),
+    );
 }
 
-export function createInterfaceMethod(doclet: IFunctionDoclet): ts.MethodSignature
-{
+export function createInterfaceMethod(
+    doclet: IFunctionDoclet,
+): ts.MethodSignature {
```

Comment:
Yeah this is happening too much, the previous format was fine.

Reply:
Same as above, I can adjust the line length after resolving other comments.